cypress/support/step-definitions/simple-color-picker-steps.js

```diff
@@ -3,79 +3,50 @@ import {
   simpleColorPickerLegendNoIFrame,
 } from "../../locators/simple-color-picker";
 import {
-  experimentalSimpleColorPickerInputInIframe,
   experimentalSimpleColorPickerInput,
+  simpleColorPicker,
 } from "../../locators/advanced-color-picker/index";
-import {
-  getKnobsInput,
-  commonDataElementInputPreviewNoIframe,
-} from "../../locators";
+import { commonDataElementInputPreviewNoIframe } from "../../locators";
 import { keyCode } from "../helper";
 
 Then("Simple Color Picker {int} element was picked up", (index) => {
   cy.wait(500);
```

Comment:
Can't we use something like cy.get "data-element="tick" instead of cy.wait(500)?

Reply:
`cy.wait` only waits for 0,5 of a second. But when we will make `cy.get(element)` cypress will/could wait for 4 sec.

Reply:
It was agreed that such things would be changed in a separate pr